### PR TITLE
feat: wallet-api custom ACRE module and handlers [LIVE-13948]

### PR DESCRIPTION
**Pull request by Justkant**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** Tested manually currently but we'll see at adding automated tests either on this PR or at a later step
- [x] **Impact of the changes:** 
  - Discover and PTX webviews, only addition so the impact should be minimal

### 📝 Description

Bootstrap the wallet-api custom ACRE module and handlers
Use hw-app-acre: https://github.com/blooo-io/hw-app-acre
Support deposit and withdraw flow for ACRE
Support bitcoin getAddress and getPublicKey
Fix Family typo
Update wallet-api deps to get new bitcoin handlers
pnpm also updated the other `package.json` files automatically when using `pnpm up -riL "@LedgerHQ/wallet-api-*"` following the pnpm overrides in the root `package.json`

### ❓ Context

- **JIRA or GitHub link**: [LIVE-13948] [LIVE-14300]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13948]: https://ledgerhq.atlassian.net/browse/LIVE-13948?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-14300]: https://ledgerhq.atlassian.net/browse/LIVE-14300?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/modals/SignTransaction/Body.tsx [hunk 1]
```diff
@@ -134,7 +136,10 @@ export default function Body({ onChangeStepId, onClose, setError, stepId, params
   } = useBridgeTransaction(() => {
     const parentAccount = params && params.parentAccount;
     const account = params && params.account;
-    const bridge = getAccountBridge(account, parentAccount);
+    const bridge = params.isACRE
+      ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (ACREBridge.accountBridge as unknown as AccountBridge<any>)
+      : getAccountBridge(account, parentAccount);
```
Comment (Justkant) [inline apps/ledger-live-desktop/src/renderer/modals/SignTransaction/Body.tsx:142]: I don't think we actually need this change as it's not doing anything special with the `hw-app-acre`

File: apps/ledger-live-mobile/src/components/Web3AppWebview/liveSDKLogic.ts [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
   account: AccountLike,
   parentAccount: Account | undefined,
   liveTx: Partial<Transaction>,
+  isACRE?: boolean,
 ): TransactionCommon {
-  const bridge = getAccountBridge(account, parentAccount);
+  const bridge = isACRE
+    ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (ACREBridge.accountBridge as unknown as AccountBridge<any>)
+    : getAccountBridge(account, parentAccount);
```
Comment (Justkant) [inline apps/ledger-live-mobile/src/components/Web3AppWebview/liveSDKLogic.ts:16]: I don't think we actually need this change as it's not doing anything special with the `hw-app-acre`

File: libs/ledger-live-common/src/bridge/useBridgeTransaction.ts [hunk 3]
```diff
@@ -240,7 +242,12 @@ const useBridgeTransaction = <T extends Transaction = Transaction>(
       statusIsPending.current = true; // consider pending until status is resolved (error or success)
 
       Promise.resolve(debounce)
-        .then(() => getAccountBridge(mainAccount, null))
+        .then(() =>
+          isACRE
+            ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              (ACREBridge.accountBridge as unknown as AccountBridge<any>)
+            : getAccountBridge(mainAccount, null),
```
Comment (Justkant) [inline libs/ledger-live-common/src/bridge/useBridgeTransaction.ts:249]: I don't think we actually need this change as it's not doing anything special with the `hw-app-acre`

**Review by ypolishchuk-ledger — APPROVED** [review]

**Review by ComradeAERGO — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]
from device experience team, looks fine 👍

**Review by kallen-ledger — APPROVED** [review]

**Review by KVNLS — APPROVED** [review]

**Review by adammino-ledger — APPROVED** [review]

**Pull request merged**